Activity URL fields and status display type (#393)
discord-userdoccers/discord-userdoccers: pages/resources/presence.mdx

| created_at ^5^       | integer                                                      | Unix timestamp (in milliseconds) of when the activity was added to the user's session  |
| session_id? ^5^      | ?string                                                      | The ID of the session associated with the activity                                     |
| platform? ^3^        | string                                                       | The [platform](#activity-platform-type) the activity is being played on                |
| supported_platforms? | array[string]                                                | The [platforms](#activity-platform-type) the activity is supported on (max 10)         |
| timestamps?          | [activity timestamps](#activity-timestamps-structure) object | Unix timestamps (in milliseconds) for start and/or end of the game                     |
| application_id?      | snowflake                                                    | The ID of the application representing the game the user is playing                    |

> | status_display_type? | ?integer                                                     | [Status display type](#status-display-type)                                            |

Reviewer: ```suggestion
| status_display_type? | ?integer                                                     | [Which field is displayed](#status-display-type) in the user's status text in the member list                                            |
```

| ------------ | ------ | -------------------------------------------------------------------------------------- |
| large_image? | string | The large [activity asset image](#activity-asset-image) (max 313 characters)           |
| large_text?  | string | Text displayed when hovering over the large image of the activity (max 128 characters) |

> | large_url?   | string | URL that is opened when clicking onthe large image (max 256 characters)                |

Reviewer: ```suggestion
| large_url?   | string | URL that is opened when clicking on the large image (max 256 characters)               |
```

| excited    | Shooting stars | Can't wait for                        |
| recommend  | Guide          | Recommendation needed                 |

###### Status Display Type

> | Value | Name    | Example                                |
> | ----- | ------- | -------------------------------------- |
> | 0     | NAME    | "Listening to Spotify"                 |
> | 1     | STATE   | "Listening to Rick Astley"             |
> | 2     | DETAILS | "Listening to Never Gonna Give You Up" |

Reviewer: ```suggestion
| Value | Name    | Description                 | Example                                |
| ----- | ------- | --------------------------- | -------------------------------------- |
| 0     | NAME    | Display the `name` field    | "Listening to Spotify"                 |
| 1     | STATE   | Display the `state` field   | "Listening to Rick Astley"             |
| 2     | DETAILS | Display the `details` field | "Listening to Never Gonna Give You Up" |
```
